includes/import/abstract-wc-product-importer.php

```diff
@@ -294,7 +280,7 @@ protected function save_product_data( $product, $data ) {
 
 		// Purchase Note.
 		if ( isset( $data['purchase_note'] ) ) {
-			$product->set_purchase_note( wc_clean( $data['purchase_note'] ) );
+			$product->set_purchase_note( $data['purchase_note'] );
```

Comment:
I think this still needs the `wc_clean`. [We use it every other time we're setting purchase note, and products/data-store do not sanitize the data](https://github.com/woocommerce/woocommerce/search?utf8=%E2%9C%93&q=set_purchase_note&type=)